umh-core/examples/example-config-comm.yaml

```
@@ -35,14 +35,13 @@ dataFlow:
+            interval: 1s # Output every 1 seconds
             count: 0 # Set to 0 for unlimited messages
         pipeline:
           processors:
             - bloblang: root = content()
         output:
-            stdout: {}
-  
-internal:
-  redpanda:
-    desiredState: stopped
+            kafka:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)